src/index.html

```diff
@@ -5,8 +5,247 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo_header">
+      <nav>
+        <div class="header">
+          <a href="#" class="home header__home">
+            <img
+              src="images/logo.png"
+              alt="logo"
+              class="home__img"
+            >
+          </a>
```

Comment:
I'm not sure whether the logo is considered a part of nav. You can try having two elements inside your header, first one being the logo and the other one - nav. Use flex or grid and space them using justify-content property.

Reply:
Yes, `nav` should include only navigation links. Logo image should be part of `header`.